src/commands/run-tests-solidus-older.yml

```diff
@@ -1,8 +1,17 @@
 description: >
   Test older releases of Solidus
 
+parameters:
+  working_directory:
+    type: string
+    default: '~/project'
+
 steps:
   - test-branch:
       branch: v3.1
+      rails_version: '~> 6.1'
```

Comment:
This will force Solidus extensions to be tested with Rails 6.x on older Solidus versions.